features/wsman/src/test/java/org/opennms/netmgt/poller/monitors/WSManMonitorTest.java

```diff
@@ -50,6 +50,7 @@
 import com.google.common.collect.Maps;
 import com.mycila.xmltool.XMLDoc;
 
+@org.junit.Ignore
```

Comment:
Is this change part of the "Add Debian build support" change? Looks like this is a different topic?

Reply:
See "A few notes" at the top of this PR  ;)

Reply:
> See "A few notes" at the top of this PR ;)

just jumped right into the files, I do better next time and read first - Sorry :)